cinema/serializers.py

```diff
@@ -1 +1,106 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Movie,
+    Genre,
+    Actor,
+    MovieSession,
+    CinemaHall,
+    Order,
+    Ticket
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name",)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
```

Comment:
The `full_name` field in `ActorSerializer` is a property, not a model field. To serialize it correctly, consider using `SerializerMethodField` and define a method to return the `full_name` value.